flutter/cocoon
# Task Attempt Summary

A list of the attempts on a Task with a link to each individual runs Stackdriver log.

## Tested
- Check that `TaskAttemptSummary` is only shown on devicelab tasks
- Check cases for attempts=0,1,2
- Check that url launcher launches expected URL

## Future Work

- Update `Task`entity to make start and endtimestamps repeatable so they can contain the start and end for each individual attempt. In general, make everything in `Task` repeatable so we can know the results of individuals runs. Once this data is available, show it in the UI.

## Preview

![task_attempt_summary](https://user-images.githubusercontent.com/2148558/70836890-99371a00-1db6-11ea-9e38-eb89bd9366c9.png)

![task_attempt_summary](https://user-images.githubusercontent.com/2148558/70837487-b10f9d80-1db8-11ea-98d9-9fd0d1fdca8e.png)

Demo: https://testchillers-dot-flutter-dashboard.appspot.com/#/build
